### DISCUSSION TITLE
Klaviyo SMS Integration not working

### DISCUSSION THREAD

**Discussion by palash-cj** [saleor/apps#329] [General]
Hi There!

I want to integrate SMS services in my project with Klaviyo, but its not working as I wish.

I tried to install it with manifest url and did the configuration as suggested in the official docs. But it is not sending event data to klaviyo dashboard from where sms should be send. Its payload has all the event data but its not saving to the klaviyo dashboard. Hence, the customer profile are being created on klaviyo but only with respective emails. There is noting in the profile except from emails. 

Also, facing challenges in sending sms after adding the phone numbers through metadata.

**Comment by lkostrowski** [comment]
Hey @palash-cj , thanks for reaching out

I think I know where is the problem and I opened an issue for that here 
https://github.com/saleor/apps/issues/348

I can't promise where it will be fixed, by we will accept the PR :)
